server/routes/boardSettings.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import boardSettings from '../controllers/boardSettingsController';
+
+const router = express.Router();
+
+router.get('/members/:id', boardSettings.getMembers);
+router.patch('/member/:id', boardSettings.updateMemberRole);
```

Comment:
Any particular reason for using HTTP.patch?

Reply:
If we want to update a particular change, in this case, it is a role. then we used patch http method